ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -18,52 +19,76 @@ import { ButtonLink, HelpIcon } from '../../StyledComponents'
 import RnaSeqJunctionOutliersTable from '../../table/RnaSeqJunctionOutliersTable'
 import { getOtherGeneNames } from '../genes/GeneDetail'
 import Transcripts from './Transcripts'
-import VariantGenes, { LocusListLabels } from './VariantGene'
+import VariantGenes, { GeneLabelContent, omimPhenotypesDetail } from './VariantGene'
 import {
   getLocus,
   has37Coords,
   Sequence,
   ProteinSequence,
   TranscriptLink,
   getOverlappedIntervals,
-  getOverlappedSpliceOutliers,
+  SPLICE_OUTLIER_OVERLAP_ARGS,
 } from './VariantUtils'
 import {
   GENOME_VERSION_37, GENOME_VERSION_38, getVariantMainTranscript, SVTYPE_LOOKUP, SVTYPE_DETAILS, SCREEN_LABELS,
 } from '../../../utils/constants'
 
-const BaseSpliceOutlierLabel = React.memo(({ variant, spliceOutliersByFamily }) => {
-  if (!spliceOutliersByFamily || spliceOutliersByFamily.length < 1) {
+const OverlappedIntervalLabels = React.memo(({ groupedIntervals, variant, getOverlapArgs, getLabels }) => {
```

Comment:
Since we are now introducing the third label shown in this component based on whether a variant overlaps an interval, I created a generic helper component to use for the existing 2 labels as well as the new one, instead of just copy-pasting and modifying to create a new componenet for the new label